### ISSUE TITLE
my dataset is mixed with color and grayscale image

### ISSUE THREAD

**Issue by shuxiao9058** [eric612/MobileNet-YOLO#18]
How to work with the mobile-yolo model?convert all the image to grayscale and then process with the model?

**Comment by shuxiao9058** [comment]
but the most dataset is grayscale，and the rgb channels are equal，the training result not well now.

I think convert all images to gray，and train with gray is best.

**Comment by eric612** [comment]
If the most dataset is grayscale , you can try to turn off image distortion , like hue,brightness,contrast,saturation_prob , these data augmentation may confuse gray images training

**Comment by shuxiao9058** [comment]
and also train with force color？

**Comment by shuxiao9058** [comment]
can i train with sigle channel(grayscale image),and maybe the training is faster.

**Comment by eric612** [comment]
Yes , I think it can get best performance between speed and accuracy as the few color images not much help training

**Comment by shuxiao9058** [comment]
yes,i will try.thanks for your reply.

**Comment by shuxiao9058** [comment]
another question, how to modify the model you provide to use grayscale for training?

does caffe has the force_gray param?

**Comment by eric612** [comment]
Sorry , I have not tried this way , you can try search same requirement as ssd caffe.
